src/components/Request/FormFields/RequestType.tsx

```diff
@@ -0,0 +1,71 @@
+import { Combobox, Label, Option, Text } from "@fluentui/react-components";
+import { DropdownIcon } from "@fluentui/react-icons-mdl2";
+import { REQUESTTYPES } from "consts/RequestTypes";
+import {
+  Control,
+  Controller,
+  FieldErrors,
+  UseFormSetValue,
+} from "react-hook-form";
+import "../Request.css";
+import { RHFRequest } from "../NewRequestForm";
+
+interface IRequestType {
+  name: string;
+  control: Control<RHFRequest, unknown>;
+  errors: FieldErrors<RHFRequest>;
+  setValue: UseFormSetValue<RHFRequest>;
+}
+
+const RequestType = (rt: IRequestType) => {
+  return (
+    <div className="requestFieldContainer">
+      <Label
+        id={rt.name + "Id"}
+        size="small"
+        weight="semibold"
+        className="requestFieldLabel"
+        required
+      >
+        <DropdownIcon className="requestFieldIcon" />
+        Request Type
+      </Label>
+      <Controller
+        name="requestType"
+        control={rt.control}
+        rules={{
+          validate: (value) => {
+            return value?.length > 0 ? undefined : "Request Type is required";
+          },
```

Comment:
Not sure I understand the complexity of adding it as a validate -- why not just required?
```suggestion
        required: "Request Type is required"
```